### PR TITLE
Respect false translationd domain

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

Similar to https://github.com/sonata-project/SonataAdminBundle/pull/7271

## Changelog

```markdown
### Fixed
- Respect `false` as translation_domain for tab and action buttons.
```

### CODE REVIEW

File: src/Resources/views/CRUD/base_edit_form.html.twig [hunk 1]
```diff
@@ -42,7 +42,12 @@
                                     {% set _tab_name = tab_prefix ~ '_' ~ loop.index %}
                                     <li{% if (not app.request.query.has('_tab') and loop.index == 1) or (tab_query_index == loop.index) %} class="active"{% endif %}>
                                         <a href="#{{ _tab_name }}" class="changer-tab" aria-controls="{{ _tab_name }}" data-toggle="tab">
-                                            <i class="fa fa-exclamation-circle has-errors hide" aria-hidden="true"></i> {{ form_tab.label|trans({}, form_tab.translation_domain ?: admin.translationDomain) }}
+                                            <i class="fa fa-exclamation-circle has-errors hide" aria-hidden="true"></i>
+                                            {% if form_tab.translation_domain is same as(false) %}
+                                                {{ form_tab.label }}
+                                            {% else %}
+                                                {{ form_tab.label|trans({}, form_tab.translation_domain ?: admin.translationDomain) }}
```
Comment (jordisala1991) [inline src/Resources/views/CRUD/base_edit_form.html.twig:49]: `form_tab.translation_domain ?? admin.translationDomain`

Does twig support the ?? operator?

Comment (jordisala1991) [reply src/Resources/views/CRUD/base_edit_form.html.twig:49]: Also on some places you use the ?:, on others ??, and others |default

IMO we should pick one.

Comment (VincentLanglet) [reply src/Resources/views/CRUD/base_edit_form.html.twig]: I tried to use ?? everywhere

File: src/Resources/views/Form/form_admin_fields.html.twig [hunk 2]
```diff
@@ -412,7 +412,7 @@ file that was distributed with this source code.
 
             {# NEXT_MAJOR: Remove this block #}
             {% if sonata_admin is defined and sonata_admin_enabled and sonata_admin.field_description.getHelp('sonata_deprecation_mute')|default(false) %}
-                <span class="help-block sonata-ba-field-help">{{ sonata_admin.field_description.help|trans(help_translation_parameters, sonata_admin.field_description.translationDomain ?: admin.translationDomain)|raw }}</span>
+                <span class="help-block sonata-ba-field-help">{{ sonata_admin.field_description.help|trans(help_translation_parameters, sonata_admin.field_description.translationDomain ?? admin.translationDomain)|raw }}</span>
```
Comment (jordisala1991) [inline src/Resources/views/Form/form_admin_fields.html.twig:415]: Do we need an if here to the case it is false the translationDomain?

Comment (VincentLanglet) [reply src/Resources/views/Form/form_admin_fields.html.twig:415]: Not sure it worth the change. This option is deprecated and the block is removed in master.

**Review by jordisala1991 — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**